polkadot/runtime/parachains/src/paras/mod.rs

```diff
@@ -1174,7 +1178,7 @@ impl<T: Config> Pallet<T> {
 		let current_block = frame_system::Pallet::<T>::block_number();
 		// Schedule the upgrade with a delay just like if a parachain triggered the upgrade.
 		let upgrade_block = current_block.saturating_add(config.validation_upgrade_delay);
-		Self::schedule_code_upgrade(id, new_code, upgrade_block, &config);
+		Self::schedule_code_upgrade(id, new_code, upgrade_block, &config, false);
```

Comment:
IMO we should "bubble up" the `set_go_ahead` parameter to the callers of this function.